tpu/hptuning/resnet-hptuning/README.md

```diff
@@ -0,0 +1,38 @@
+
+## Introduction
+
+This sample shows how to run hyperparameter tuning jobs on Cloud Machine Learning Engine with Cloud TPUs using TensorFlow's `tf.metrics`.
```

Comment:
Not sure it's needed to mention "..using TF tf.metrics"

Reply:
This is to contrast the other possible ways of reporting metrics (`tf.summary`, and `cloudml.hypertune`).